app/src/main/java/org/mozilla/fenix/settings/about/AboutFragment.kt

```diff
@@ -0,0 +1,184 @@
+
+/**
+ * Displays the logo and information about the app, including library versions.
+ */
+class AboutFragment : Fragment() {
+    private lateinit var store: AboutPageStore
+    private lateinit var appName: String
+    private lateinit var aboutView: AboutView
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val rootView = inflater.inflate(R.layout.fragment_about, container, false)
+        appName = getString(R.string.app_name)
+        activity?.title = getString(R.string.preferences_about, appName)
+
+        store = AboutPageStore(initializeState())
+        val controller = DefaultAboutFragmentController(
+            metrics = context?.metrics,
+            whatsNewViewed = { WhatsNew.userViewedWhatsNew(context!!) },
+            openToCustomTab = ::openLinkInCustomTab,
+            openLibraries = ::openLibrariesPage
+        )
+        val interactor = AboutPageInteractor(controller)
+        aboutView = AboutView(rootView.about_layout, interactor)
+
+        return rootView
+    }
+
+    @ExperimentalCoroutinesApi
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        consumeFrom(store) {
+            aboutView.update(it)
+        }
+    }
+
+    private fun initializeState(): AboutPageState {
+        val items: MutableList<AboutPageItem> = mutableListOf()
+
+        items.add(populateAboutHeader())
+        items.addAll(populateAboutList())
+
+        return AboutPageState(items)
+    }
+
+    private fun populateAboutHeader(): AboutPageItem.Header {
+        val aboutText = try {
+            val packageInfo = requireContext().packageManager.getPackageInfo(requireContext().packageName, 0)
+            val versionCode = PackageInfoCompat.getLongVersionCode(packageInfo).toString()
+            val componentsVersion = mozilla.components.Build.version + ", " + mozilla.components.Build.gitHash
+            val maybeGecko = if (SDK_INT >= Build.VERSION_CODES.N) GECKO_EMOJI else "GV"
+            val geckoVersion = GeckoViewBuildConfig.MOZ_APP_VERSION + "-" + GeckoViewBuildConfig.MOZ_APP_BUILDID
+
+            String.format(
+                "%s (Build #%s)\n%s: %s\n%s: %s",
+                packageInfo.versionName,
+                versionCode,
+                COMPONENTS_EMOJI,
+                componentsVersion,
+                maybeGecko,
+                geckoVersion
+            )
+        } catch (e: PackageManager.NameNotFoundException) {
+            ""
+        }
+
+        val content = getString(R.string.about_content, appName)
+        val buildDate = BuildConfig.BUILD_DATE
+
+        return AboutPageItem.Header(aboutText, content, buildDate)
+    }
+
+    private fun populateAboutList(): List<AboutPageItem> {
+        val myList: MutableList<AboutPageItem> = mutableListOf()
+
+        context?.let {
```

Comment:
We probably want to crash here if context isn't found, rather that displaying an empty list. `getString` in the previous method will crash anyways.